tests/test_multigpu.py

```diff
@@ -66,23 +70,16 @@ def test_distributed_data_loop(self):
         with patch_environment(omp_num_threads=1, cuda_visible_devices="0,1"):
             execute_subprocess_async(cmd, env=os.environ.copy())
 
-    # Need to see why this test raises forking issues when ran as a suite
-    @skip
```

Comment:
Essentially the `notebook_launcher` doesn't like using async subprocesses when doing things, so executing it fully as a non-async subprocess makes this fully work reliably.